helm_deploy/hmpps-remand-and-sentencing-api/values.yaml

```diff
@@ -29,6 +29,11 @@ generic-service:
   namespace_secrets:
     hmpps-remand-and-sentencing-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+    rds-postgresql-instance-output:
```

Comment:
I think you might need `DB_SSL_MODE: "verify-full"` in env and `?sslmode=${DB_SSL_MODE}` at the end of the datasource url in application.yml